debian/clearwater-socket-factory.prerm

```diff
@@ -52,6 +52,12 @@ set -e
 
 case "$1" in
     remove|upgrade|deconfigure)
+        # Stop and deconfigure the management and signaling upstart jobs
+        stop clearwater-socket-factory-mgmt > /dev/null 2>&1 || true
+        stop clearwater-socket-factory-sig > /dev/null 2>&1 || true
```

Comment:
Can you double check that the prerm is called before the postinst when upgrading a package. If not the services will end up being stopped permanently!